pages/search/search.jsx

```diff
@@ -21,14 +21,7 @@ class Search extends React.Component {
   }
 
   componentDidMount() {
-    // The focus() function of <input /> isn't exposed by <DebounceInput />
```

Comment:
Avoid accessing/manipulating DOM directly in React unless you really don't have a way to access element within the React component

Reply:
Agreed, I didn't know that issue in your comment was closed...I should've checked.